travis.sh

```diff
@@ -179,25 +181,32 @@ function prepare_ros_workspace() {
          http://* | https://* | file://*) ;; # use url as is
          *) item="file://$CI_SOURCE_PATH/$item" ;; # turn into proper url
       esac
-      travis_run_true wstool merge -k $item
+      travis_run_true curl -s -o $upstream_workspace_file $item
```

Comment:
Doesn't this overwrite `upstream_workspace_file` for each item in the loop and thus only the last one is kept?

Reply:
You're right, it probably makes more sense to import the files right away. `vcs` doesn't seem to support importing from urls

Reply:
Just `curl`-download to a file in `/tmp` and `vcs import` from there.